packages/readline.rb

```diff
@@ -1,32 +1,30 @@
-require 'package'
+equire 'package'
 
 class Readline < Package
   description 'The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in.'
   homepage 'http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html'
-  version '8.0'
+  version '8.1'
   compatibility 'all'
-  source_url 'https://ftpmirror.gnu.org/gnu/readline/readline-8.0.tar.gz'
-  source_sha256 'e339f51971478d369f8a053a330a190781acb9864cf4c541060f12078948e461'
+  source_url 'https://gnu.askapache.com/readline/readline-8.1.tar.gz'
+  source_sha256 'f8ceb4ee131e3232226a17f51b164afc46cd0b9e6cef344be87c65962cb82b02'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.0-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.1-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.1-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/readline-8.1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'd11d2b5e5180d2752b8e56696865daa60761cb5b93e27706205399baa13bbbb3',
-     armv7l: 'd11d2b5e5180d2752b8e56696865daa60761cb5b93e27706205399baa13bbbb3',
-       i686: '1dc734cf28063f2b552733cdb04219f7de76f3b59feab84ffe987a421e01a24c',
-     x86_64: '19575599c41f80d4fd0753a25f0591952c4eb4c552768432064605d195120626',
+     aarch64: '3df33883be2751b151e39c3e17905104002b078fd4e53f11048649c8a34a796f',
+      armv7l: '3df33883be2751b151e39c3e17905104002b078fd4e53f11048649c8a34a796f',
+        i686: '15526e871c958dff346de263d40775be55b00a248599dab227e3ebab39e0ac1f',
+      x86_64: 'bb4452115aca711846392413c50093f813ce68b112780dc7807446f446ab49a4',
   })
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-static',
-           '--with-curses'
+    system "./configure #{CREW_OPTIONS} \
+    --disable-install-examples \
+    --with-curses"
```

Comment:
Does this break anything?  There are so many packages that depend on this.

Reply:
Better safe than sorry. I'll take that line out.

Reply:
Probably a better solution would be to recompile other packages if something breaks.  More work, I know but we don't want to sacrifice capabilities, right?

Reply:
yup. Updated with new uploads.